docs/public-networks/how-to/bonsai-limit-trie-logs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Limit Trie Logs for Bonsai
+sidebar_position: 12
+description: Enable this early access feature to reduce the size of your database
+tags:
+  - public networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Limit Trie Logs for Bonsai
```

Comment:
Bonsai... to reduce database size.

change this for SEO

Reply:
Are you suggesting, "Limit Trie Logs to Reduce Database Size"?

Are you happy to exclude Bonsai from the title? I wanted to make it clear it was irrelevant for FOREST, but if we plan to remove FOREST soon, maybe that doesn't matter.